### PR TITLE
changed to new syntax

### PR DESCRIPTION
**Pull request by richardwhateverr**

### CODE REVIEW

File: java/lernjava.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ alias: ["lern-java"]
...
+Es scheint so, als würdest du ziemlich oft ziemlich einfache Fragen stellen, welche ganz einfach durch einen Blick in die Docs (`/t spigot-docs`) oder durch das Kennenlernen der Java-Grundlagen (`/t java-lernen`) lösbar wären. Bitte erwarte nicht, dass wir dir hier alles vorkauen und dir dauerhaft einfach Code zum kopieren bereitstellen, denn wir sind nicht deine Sklaven. Lerne dein Zeug selber, damit mehr Platz für wirklich sinnvolle Fragen besteht, denn alles andere kostet nur unnötig Zeit und Nerven. Versteh' mich nicht falsch. In <#486919573229010944> sind solche Fragen sogar erwünscht (bitte aber wirklich nur Java an sich, kein Minecraft-Gedöns), da dort klar wird, dass du **Java** lernst und dich bemühst, selbst etwas für deinen Erfolg zu tun, anstatt auf ein möglichst schnelles Ergebnis im **Spigot**-Bereich zu hoffen, um dich möglichst schnell "Minecraft-Dev" nennen zu können.
 
 Fazit: Lern Java, bevor du mit Spigot anfängst!
 
-**Tipp:** Brauchst du Hilfe beim Suchen? `st suchen` sollte dir helfen!
+**Tipp:** Brauchst du Hilfe beim Suchen? `/t suchen` sollte dir helfen!
```
Comment (daspositioned) [inline java/lernjava.md]:
```suggestion
**Tipp:** Brauchst du Hilfe beim Suchen? `/tag suchen` sollte dir helfen!
```

File: java/lernjava.md [hunk 2]
```diff
@@ -5,8 +5,8 @@ alias: ["lern-java"]
...
+Es scheint so, als würdest du ziemlich oft ziemlich einfache Fragen stellen, welche ganz einfach durch einen Blick in die Docs (`/t spigot-docs`) oder durch das Kennenlernen der Java-Grundlagen (`/t java-lernen`) lösbar wären. Bitte erwarte nicht, dass wir dir hier alles vorkauen und dir dauerhaft einfach Code zum kopieren bereitstellen, denn wir sind nicht deine Sklaven. Lerne dein Zeug selber, damit mehr Platz für wirklich sinnvolle Fragen besteht, denn alles andere kostet nur unnötig Zeit und Nerven. Versteh' mich nicht falsch. In <#486919573229010944> sind solche Fragen sogar erwünscht (bitte aber wirklich nur Java an sich, kein Minecraft-Gedöns), da dort klar wird, dass du **Java** lernst und dich bemühst, selbst etwas für deinen Erfolg zu tun, anstatt auf ein möglichst schnelles Ergebnis im **Spigot**-Bereich zu hoffen, um dich möglichst schnell "Minecraft-Dev" nennen zu können.
```
Comment (daspositioned) [inline java/lernjava.md]:
```suggestion
Es scheint so, als würdest du ziemlich oft ziemlich einfache Fragen stellen, welche ganz einfach durch einen Blick in die Docs (`/tag spigot-docs`) oder durch das Kennenlernen der Java-Grundlagen (`/tag java-lernen`) lösbar wären. Bitte erwarte nicht, dass wir dir hier alles vorkauen und dir dauerhaft einfach Code zum kopieren bereitstellen, denn wir sind nicht deine Sklaven. Lerne dein Zeug selber, damit mehr Platz für wirklich sinnvolle Fragen besteht, denn alles andere kostet nur unnötig Zeit und Nerven. Versteh' mich nicht falsch. In <#486919573229010944> sind solche Fragen sogar erwünscht (bitte aber wirklich nur Java an sich, kein Minecraft-Gedöns), da dort klar wird, dass du **Java** lernst und dich bemühst, selbst etwas für deinen Erfolg zu tun, anstatt auf ein möglichst schnelles Ergebnis im **Spigot**-Bereich zu hoffen, um dich möglichst schnell "Minecraft-Dev" nennen zu können.
```

**Review by daspositioned — CHANGES_REQUESTED** [review]
The new tag bot Krile uses /tag, not /t

**Review by daspositioned — DISMISSED** [review]
Needs to be fixed in other tags as well, but this looks good to me now. Maybe we can do it all in one PR?

**Comment by richardwhateverr** [comment]
> Needs to be fixed in other tags as well, but this looks good to me now. Maybe we can do it all in one PR?

Done

**Review by daspositioned — APPROVED** [review]

**Review by rainbowdashlabs — APPROVED** [review]

**Pull request merged**